pkg/server/server.go

```diff
@@ -96,19 +96,30 @@ func renderer(templatePath string) multitemplate.Renderer {
 	return r
 }
 
-func requireAuth(db *sqlx.DB, secret string) func(*gin.Context) {
+func jobAuth(ctx *gin.Context, db *sqlx.DB, secret string) (string, error) {
+	jobID := ctx.Param("id")
+	job, err := data.GetJob(jobID, db)
+	if err != nil {
+		log.Println(fmt.Errorf("requireAuth failed to getJob: %w", err))
```

Comment:
Should reflect the name of this function

Reply:
Haha, the dangers of copy and paste. I'll get that fixed.